binding/vue/src/porcupine_types.ts

```diff
@@ -0,0 +1,64 @@
+import { WebVoiceProcessor } from '@picovoice/web-voice-processor';
+
+/**
+ * Type alias for the Porcupine keywords.
+ */
+export type PorcupineKeyword = PorcupineKeywordCustom | PorcupineKeywordBuiltin;
+
+/**
+* Type alias for builtin keywords.
+*/
+export type PorcupineKeywordBuiltin = {
+  builtin: string;
+  sensitivity?: number;
+};
+ 
+/**
+* Type alias for custom keywords.
+*/
+export type PorcupineKeywordCustom = {
+  base64: string;
+  custom: string;
+  sensitivity?: number;
+};
+ 
+/**
+* Type alias for PorcupineWorkerFactory arguments.
+*/
+export type PorcupineWorkerFactoryArgs = {
+  accessKey: string;
+  keywords: Array<PorcupineKeyword | string> | PorcupineKeyword | string;
+  start?: boolean;
+};
+ 
+/**
+* The language-specific worker factory, imported as { PorcupineWorkerFactory } from the 
+* @picovoice/porcupine-web-xx-worker series of packages, where xx is the two-letter language code.
+*/
+export interface PorcupineWorkerFactory extends Object {
+  create: (
+    accessKey: string, 
+    keywords: Array<PorcupineKeyword | string> | PorcupineKeyword | string,
+    keywordDetectionCallback?: CallableFunction,
```

Comment:
it has nothing to do with this PR, but i noticed we have this `CallableFunction`s everywhere and I think it's better to make them more specific at some point, something like `(keywordLabel: string) => void`

Reply:
Sure we should do this later on.